### ISSUE TITLE
Copy command fails for the very first build.

### ISSUE THREAD

**Issue by golden96371** [saucecontrol/InheritDoc#10]
Hello,

I've encountered an issue with the version 1.3.0 of this package. When I create a new project and reference the package before the first build, the build fails when copying the (not yet existing) XML file.

`MSB3030: Could not copy the file 'obj\Debug\net6.0\ClassLibrary1.xml' because it was not found.`

Here the code of the ClassLibrary1.csproj:
```
<Project Sdk="Microsoft.NET.Sdk">

  <PropertyGroup>
    <TargetFramework>net6.0</TargetFramework>
    <ImplicitUsings>enable</ImplicitUsings>
    <Nullable>enable</Nullable>
    <GenerateDocumentationFile>True</GenerateDocumentationFile>
    <InheritDocEnabled>True</InheritDocEnabled>
  </PropertyGroup>

  <ItemGroup>
    <PackageReference Include="SauceControl.InheritDoc" Version="1.3.0">
      <PrivateAssets>all</PrivateAssets>
      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
    </PackageReference>
  </ItemGroup>

</Project>
```

Of course I can workaround this by checking the existence of this file before setting `InheritDocEnabled`. But ideally the package can handle this by itself. Or did I miss any option?

Thanks

**Comment by saucecontrol** [comment]
There's a check in place already that should ensure the InheritDoc task doesn't run until after the compiler has generated the XML file, but I suppose a race condition isn't out of the question.

https://github.com/saucecontrol/InheritDoc/blob/b6933b0cad00a1aee62106ecec39ff74e36c7f12/src/InheritDoc/package/build/SauceControl.InheritDoc.targets#L12

I'll try out your repro later and see if it works for me.  Can you tell me what .NET SDK version you're running?  A binlog (`dotnet build -bl`) would also be useful if you can share it.

**Comment by saucecontrol** [comment]
Interesting.  I haven't been able to reproduce the error at all here.

By sharing the binlog, I meant the `msbuild.binlog` file that is generated when you use the `-bl` switch.  There's a viewer [here](https://msbuildlog.com/) that parses the log and allows you to trace what's actually happening in MSBuild.  If you can attach one from a build that errors, I'd be able to diagnose it without having to get a failed build myself.

**Comment by golden96371** [comment]
I wasn't able to reproduce the error in the command line using the -bl flag until now. Even in Visual Studio, it only occurs randomly. I will keep you updated as soon as I get a useful binlog.

**Comment by saucecontrol** [comment]
I finally realized what was happening here.  There was a check that automatically disabled InheritDoc for [design-time builds](https://github.com/dotnet/project-system/blob/main/docs/design-time-builds.md), but it only took effect if the `InheritDocEnabled` value wasn't explicitly set in the project.

I've fixed that in https://github.com/saucecontrol/InheritDoc/commit/4c4d760073236263de26708f929a1a67f21b251b, which will be part of the upcoming 2.0 release.  That should not only allow you to remove your workaround but should also improve your VS responsiveness (design-time builds run quite frequently).
